package.json

```diff
@@ -13,17 +13,25 @@
     "url": "https://bugzilla.mozilla.org/"
   },
   "scripts": {
-    "lint": "./node_modules/.bin/eslint kuma/",
+    "stylelint": "./node_modules/.bin/stylelint \"kuma/static/styles/**/*.scss\"",
+    "eslint": "./node_modules/.bin/eslint kuma/",
     "optim:svg": "svgo jinja2/includes/icons/* --disable={removeViewBox,removeTitle} --multipass --pretty",
-    "precommit": "npm run lint"
+    "precommit": "npm run eslint && npm run stylelint"
+  },
+  "dependencies": {
+    "clean-css": "3.4.28",
+    "fibers": "2.0.2",
```

Comment:
As far as I can tell, `fibers` is not used/needed (even as a dependency), and we should remove it. I tested this by removing it from `package.json`, re-generating the `package-lock.json`, re-building the `kuma_base` image, bringing-up a fresh docker-compose (`down` then `up -d`), and then testing everything again(`npm run stylelint`, `npm run eslint`, `make build-static`, `DJANGO_SETTINGS_MODULE=kuma.settings.prod make build-static`, etc.). All of the node tools we use in Kuma worked fine without it.

Also, now that we're on Node.js 8 and the nice new world of `async` and `await`, I don't think we should encourage anyone to reach for `fibers`, which is most often used as a way to write asynchronous code that looks like it's synchronous.